CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+# Welcome to openshift-python-wrapper contributing guide
```

Comment:
It looks good. Examples are to the point but helpful. 

I think it would be good to add some more information for people outside the organization/openshift-virtualization working with this repo:

1) Whats  is the process to follow when someone finds a bug? or needs to enhance a module? Should they directly open a PR or discuss with the maintainers first? Or start with logging an issue?
2) How do they reach the maintainers for questions?

These are not really a problem for us, but for someone not part of same org/starting new, it may not be easy to figure it out.

Also, if there is any requirement from the PR itself, it would be good to have it documented here. e.g. description field is added to a release, and it should be concise and clear one line summary. 

For a new resource, is there any requirement of associated docs?

Reply:
Added a section about `issues` and `pull requests`, please let me know what you think

Reply:
`For a new resource, is there any requirement of associated docs?`  see `Add a link to the resource's API reference` - do you think we need more?

Reply:
Thank you for adding the sections. Both looks great! For the new resource, `Add a link to the resource's API reference` is good enough. I also think it might be good to have information on doc header styles, if anyone decides to add any. This is relevant for all codes though.